example/env.d/learninglocker

```diff
@@ -0,0 +1,26 @@
+# Misc
```

Comment:
Why about naming this file `env.d/learninglocker.example`? And adding a make rule to enable (copy) it?

Reply:
It is in an `example` directory, I thoughr it should be enough to understand that it is an... example ?

Reply:
Done